### ISSUE TITLE
[studio] Upgrade manager to add `<additionalFields />` to content type definitions that have date-time and time controls

### ISSUE THREAD

**Issue by rart** [craftercms/craftercms#5239]
#### Is your feature request related to a problem? Please describe.
Upgrade manager to add <additionalFields /> to all content-type definitions that have a `date-time` or `time` controls.
It should include an <id> entry with the value `{fieldId}_tz`.

#### Describe the solution you'd like
```xml
<field>
    <type>date-time</type>
    <id>dateTime_dt</id>
    <title>Flush</title>
    ...
    <additionalFields>
      <id>dateTime_dt_tz</id> <!-- field id + '_tz' -->
    </additionalFields>
</field>
```

#### Describe alternatives you've considered
n/a

Related to https://github.com/craftercms/craftercms/issues/5083

**Comment by rart** [comment]
@jdrossl I wasn't sure how to test this, but it didn't seem to work.

I had a few sites prior to this update, so all I did was upgrade my crafter and start it up. Looking at the git log, I do see _"Site upgrade from v3.1.9 to v3.1.18: Update date-time & time controls with addititional fields"_ however, the content type that had a date-time and time fields did not get the `additionalFields` tag.

I don't see how it would matter but I did format the XML and committed the change while the server was down. I'm attaching my site sandbox for you to inspect: [sandbox.zip](https://github.com/craftercms/craftercms/files/7869145/sandbox.zip)

Creating a new site "works" but that's just the blueprint having the additionalFields tag already there.
